easybuild/easyconfigs/q/Qhull/Qhull_pkgconfig.patch

```diff
@@ -0,0 +1,18 @@
+inspired by qhull-2003.1-pkgconfig.patch, available from git://pkgs.fedoraproject.org/qhull
```

Comment:
why not simplify add it to the `configure.ac`  and let that take care of everything?